views/2023/social-events.haml

```diff
@@ -0,0 +1,36 @@
+.introduction
+  .container-full.mb-4
+    %h3.event-title An array of fun and varied activities to surround the conference!
+
+.events
+  .container-full
+    .row.mb-5
+      .col-lg-3.col-md-4
+        .avatar.social-event-avatar
+          %img{ src: '/2023/images/alto.jpg' }
+      .col-lg-9.col-md-8
+        .details
+          %h3.event-title Welcome Games Night
+          .description
+            %p To kick things off we have organised unlimited games at King Pin Crown for 2-hours to celebrate the start of the conference and the Ruby Community. Finger Food and drinks will be provided.
+            %p This event is free for all conference attendees.
+          .location
+            Wednesday 19th February, 7pm-9:30pm <br />
+            %a{ href: 'https://goo.gl/maps/nMorFH1wGMq1mSUP8' }
+              %i.fa.fa-map-marker-alt
+              King Pin - Upstairs at Crown Casino - 8 Whiteman Street, Melbourne
+    .row.mb-5
+      .col-lg-3.col-md-4
+        .avatar.social-event-avatar
+          %img{ src: '/2023/images/boatbuilders-yard.jpg' }
```

Comment:
I can see `boatbuilders-yard.jpg` in `assets/2023/images` but don't understand why it isn't loading.

Reply:
loading for me now in my local.